public/css/lobby.css

```diff
@@ -0,0 +1,62 @@
+#body {
+    min-height: 100%;
+    min-width: 100%;
+}
+
+ul{
+    overflow: hidden;
```

Comment:
Isn't it `overflow-x` ?

Reply:
Actually, both should be hidden. The css isn't meant to display a LOT of players.